zypp/misc/CheckAccessDeleted.h

```diff
@@ -82,20 +85,23 @@ namespace zypp
        */
       size_type check( bool verbose_r = false );
 
-      bool empty() const		{ return _data.empty(); }
-      size_type size() const		{ return _data.size(); }
-      const_iterator begin() const	{ return _data.begin(); }
-      const_iterator end() const	{ return _data.end(); }
+      size_type check( const Pathname &lsofOutput, bool verbose_r = false );
```

Comment:
`/** \overload and a few words about the role of \a lsofOutput in this overloaded method. */`